make/verify-dependencies.mk

```
@@ -0,0 +1,12 @@
+.PHONY: verify-dependencies
+## Runs commands to verify after the updated dependecies of toolchain-common/API(go mod replace), if the repo needs any changes to be made
+verify-dependencies: tidy generate vet build test lint
```

for this particular case, we should use go lint only
```suggestion
verify-dependencies: tidy generate vet build test lint-go-code
```
also, I don't think that the yaml files are lint-compatible right now

---

Yes.. i figured when i was testing it locally, will change it .

---

updated it

---

yeah, as suggested only `lint-go-code` , the yaml linter might fail I've observed.